app/src/js/components/button.jsx

```diff
@@ -1,21 +1,42 @@
 'use strict';
 
 const { React, Base } = require('./base');
+const ColorPicker = require('react-color');
+const sweetAlert = require('sweetalert');
+
+module.exports = class Component extends Base {
+
+  constructor() {
+
+    super();
```

Comment:
...`super(props)` here, otherwise any props set on component won't work. Always set a React component constructor this way in ES6.